cosmos/api.py

```diff
@@ -15,6 +15,7 @@
 import funcsigs
 import re
 
+from black_magic.decorator import partial
```

Comment:
@egafni this was removed in a previous release but many cosmos clients including PIPE still depend on it. Should they import black_magic themselves? Or is there an alternate implementation you recommend?